setup/install/providers/aws/aws-ec2.md

```diff
@@ -93,6 +96,25 @@ Option number 1 is useful for creation of user with AWS Access Key and secret. T
 2. Using AWS IAM Roles
 Option 2 uses the IAM roles *ManagingRole* and *ManagedRoles*. This setting is applied on some environments that have extra security considerations.
 
+### Spinaker AssumeRole Policy
+1. Incase, if you miss to add append the Line 158, then Manually create a Policy by Navigating to **Console > IAM > Policies**.
```

Comment:
I saw your reponse to this before, @anooptejt . Sorry, I'm still not getting it. 
What does "miss to add append the line" mean? Do we mean if I _choose_ not to add another managed account to the `SpinnakerAssumeRolePolicy` section? 

Or, under what circumstances exactly would I manually create a policy?

Reply:
This manual Policy we can add it as TroubleShooting. I have noticed clouddriver was not starting up as expected. Hence, i have manually created a policy and checked it out. It worked like a charm. So I would recommend to add this as a Troubleshooting item. Please suggest in case of any other thoughts.

Reply:
I'll let @clareliguori weigh in on this, for technical input, then I'll look at the wording. Thanks, @anooptejt!

Reply:
It would be better to have instructions on how to add a new managed account post-stack-creation by adding it to the template in the SpinnakerAssumeRolePolicy section and then updating the existing CloudFormation stack.

Reply:
In the cloudformation template already there is a section to add the SpinnakerAssumeRolePolicy. However, for some weird reason that doesnt make any updates in the AWS. Which is why I have added this section explicitly in the document. Which would make it more easier for users to enable AWS with Spinnaker.

Reply:
Can you provide repro steps where adding a new account to the list in the SpinnakerAssumeRolePolicy section in the template does not cause updates in AWS?

Reply:
Search for SpinnakerAssumeRolePolicy in the managing.yaml
Add the managed account and tried to create the stack using the template. I got SpinnakerAssumeRolePolicy created empty without any of the attributes defined in the template. I had to manually add the #arn in the below format

arn:aws:iam::YOUR_MANAGED_ACCOUNT1:role/spinnakerManaged

Reply:
I tried to reproduce with the following steps, but I wasn't able to - it worked as expected:

```
# Deploy the stack without updating the template first
$ aws cloudformation deploy \
	--stack-name test-spinnaker-managing-infrastructure-setup \
	--template-file managing.yaml \
	--parameter-overrides UseAccessKeyForAuthentication=true \
	--capabilities CAPABILITY_NAMED_IAM 

# Find the IAM user name
$ aws cloudformation describe-stack-resources \
	--stack-name test-spinnaker-managing-infrastructure-setup \
	--logical-resource-id SpinnakerUser

# Output the provisioned user policy
$ aws iam get-user-policy --policy-name SpinnakerAssumeRolePolicy --user-name test-spinnaker-managing-infrastructu-SpinnakerUser-1V1NBNV5OEEV4
{
    "UserName": "test-spinnaker-managing-infrastructu-SpinnakerUser-1V1NBNV5OEEV4",
    "PolicyName": "SpinnakerAssumeRolePolicy",
    "PolicyDocument": {
        "Version": "2012-10-17",
        "Statement": [
            {
                "Action": "sts:AssumeRole",
                "Resource": [
                    "arn:aws:iam::XXXXXXXXXXXX:role/spinnakerManaged"
                ],
                "Effect": "Allow"
            }
        ]
    }
}

# Updated managing.yaml template with new managed account
      PolicyDocument:
        Version: "2012-10-17"
        Statement:
          - Action: sts:AssumeRole
            Effect: Allow
            Resource:
              - !Sub arn:aws:iam::${AWS::AccountId}:role/spinnakerManaged # This is the current account
              - arn:aws:iam::YYYYYYYYYYYY:role/spinnakerManaged # New Managed Account
      PolicyName: SpinnakerAssumeRolePolicy

# Update the stack with the updated template that contains a new managed account
$ aws cloudformation deploy \
	--stack-name test-spinnaker-managing-infrastructure-setup \
	--template-file managing.yaml \
	--parameter-overrides UseAccessKeyForAuthentication=true \
	--capabilities CAPABILITY_NAMED_IAM 

# Output the updated provisioned user policy
$ aws iam get-user-policy --policy-name SpinnakerAssumeRolePolicy --user-name test-spinnaker-managing-infrastructu-SpinnakerUser-1V1NBNV5OEEV4
{
    "UserName": "test-spinnaker-managing-infrastructu-SpinnakerUser-1V1NBNV5OEEV4",
    "PolicyName": "SpinnakerAssumeRolePolicy",
    "PolicyDocument": {
        "Version": "2012-10-17",
        "Statement": [
            {
                "Action": "sts:AssumeRole",
                "Resource": [
                    "arn:aws:iam::XXXXXXXXXXXX:role/spinnakerManaged",
                    "arn:aws:iam::YYYYYYYYYYYY:role/spinnakerManaged"
                ],
                "Effect": "Allow"
            }
        ]
    }
}

```